packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -121,12 +121,35 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
 
     const decrementSlotProps = computed(() => ({ click: onClickDown }))
 
+    // Control holding down states
+    let hldDwnReqNextTime = 0
+    let hldDwnReq = -1
+    const hldDwnReqDelay = 100
+    const hldDwnDelay = 500
+    let hldDwn: null | 'up' | 'down' = null
+    let hldDwnTimeout = -1
+
+    function controlHoldingDownWatcher (time: number) {
+      hldDwnReq = requestAnimationFrame(controlHoldingDownWatcher)
```

Comment:
- `hldDwnReq` » `holdDownRaf` or `holdActionRaf`

I actually don't care much about the suffix. It can be `*Raf`,  `*Frame`,  `*Req`. My issue is with saving 2 vowels for no obvious gain.